### PR TITLE
Reorganized example isolators.

### PR DESCRIPTION
**Pull request by ConnorDoyle**
- Update OIR lib to get node name from env.

### CODE REVIEW

**Comment by ConnorDoyle** [comment]
Updated during demo prep

**Comment by ConnorDoyle** [comment]
Demo docs (wip): https://gist.github.com/ConnorDoyle/ddfa508473de5c77a0fc931b13f1ff49

File: cluster/addons/iso-client/examples/cpuset-visualizer-0.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```
Comment (squall0gd) [inline cluster/addons/iso-client/examples/cpuset-visualizer-0.yaml:1]: Add kubernetes header and information about purpose of this file.

File: cluster/addons/iso-client/examples/cpuset-visualizer-1.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```
Comment (squall0gd) [inline cluster/addons/iso-client/examples/cpuset-visualizer-1.yaml:1]: Add kubernetes header and information about purpose of this file.

File: cluster/addons/iso-client/examples/cpuset-visualizer-2.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```
Comment (squall0gd) [inline cluster/addons/iso-client/examples/cpuset-visualizer-2.yaml:1]: Add kubernetes header and information about purpose of this file.

**Comment by flyingcougar** [comment]
i was thinking about approach with go lang `pkg/` and `src/` but in general your approach is easier and cleaner (less "directory" overhead).

**Review by flyingcougar — CHANGES_REQUESTED** [review]
Should we have `Makefile` and `Dockerfile` for noop as well ?

**Pull request closed without merging**